Library/Homebrew/cmd/update.rb

```diff
@@ -115,6 +163,12 @@ def load_tap_migrations
   rescue LoadError
     false
   end
+
+  def load_formula_renames
+    load 'formula_renames.rb'
```

Comment:
I think there's no need to change it to 'load' because `tap_migrations.rb` is loaded only once and only in `update`.